docs/samples/v1beta1/transformer/torchserve_image_transformer/image_transformer/__main__.py

```diff
@@ -18,22 +18,29 @@
 from .transformer_model_repository import TransformerModelRepository
 
 DEFAULT_MODEL_NAME = "model"
+CONFIG_PATH = "/mnt/models/config/config.properties"
 
 parser = argparse.ArgumentParser(parents=[kserve.model_server.parser])
 parser.add_argument(
     "--predictor_host", help="The URL for the model predict function", required=True
 )
+parser.add_argument(
+    "--config_path", help="The path to the config file containing the list of model names",
+    required=False, default=CONFIG_PATH
+)
+parser.add_argument(
```

Comment:
you can add `type=bool` so you do not need to parse strings

Reply:
actually try `parser.add_argument('--log_latency',  default=False, action='store_true')`

Reply:
discussed offline: this is user implementation so we still want to validate. and truthy python values are annoying !